### PR TITLE
fix geth.log on Android

### PR DESCRIPTION
**Pull request by rasom**
log file can be found in `Download` folder

On android 5 file will be there right after starting application. On Android 6+ you will need to provide `Storage` permission in app's settings (Settings->Application manager->Status->Permissions->Storage).

status: ready

### CODE REVIEW

**Comment by jeluard** [comment]
@rasom What if you don't have a sdcard?

**Review by oskarth — APPROVED** [review]
> @rasom What if you don't have a sdcard?
I wonder about this too

File: modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java [hunk 1]
```diff
@@ -180,10 +180,13 @@ private void doStartNode(final String defaultConfig) {
             JSONObject customConfig = new JSONObject(defaultConfig);
             JSONObject jsonConfig = new JSONObject(config);
             String gethLogFileName = "geth.log";
+            String dataDir = root + customConfig.get("DataDir");
```
Comment (oskarth) [inline modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java]: Is this always present?

Comment (rasom) [reply modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java]: That's not something introduced in this PR, i only moved it to variable

**Comment by rasom** [comment]
Right now we don't store logs to `/sdcard` but to `Environment.getExternalStorageDirectory().getAbsolutePath()`, and i can see logs on galaxy note 4. But want to verify with @flexsurfer because he still can't see logs on the device.

Will update PR description later

**Review by yenda — APPROVED** [review]

**Comment by asemiankevich** [comment]
----------------
Branch: PR-3077
Android: https://i.diawi.com/xtmFYn
iOS: https://i.diawi.com/JK1XgH

I can find the geth.log file after i grant status app with permission to acccess storage
http://prntscr.com/i5cc3s

**Pull request merged**